packages/pay/test/setup.spec.ts

```diff
@@ -55,30 +55,41 @@ describe('open payments', () => {
 
     const { ilpAddress, sharedSecret } = streamServer.generateCredentials()
 
-    const invoiceId = uuid()
+    const incomingPaymentId = uuid()
     const accountUrl = 'https://wallet.example/alice'
-    const invoiceUrl = `${accountUrl}/invoices/${invoiceId}`
+    const destinationPayment = `${accountUrl}/incoming-payments/${incomingPaymentId}`
     const expiresAt = Date.now() + 60 * 60 * 1000 * 24 // 1 day in the future
     const description = 'Coffee'
+    const externalRef = ''
+    const receiptsEnabled = false
 
     nock('https://wallet.example')
```

Comment:
These `nock`s could be moved to a single setup function that takes optional incoming payment parameters specific to each test.
For reference:
https://github.com/interledger/rafiki/blob/main/packages/backend/src/webhook/service.test.ts#L107